index.js

```diff
@@ -148,18 +148,25 @@ async function cloneExample() {
     await run(`git clone --depth 1 --filter=blob:none --sparse ${config.examples.repoUrl} ${tmpDir}`);
     // Checkout just the example dir.
     await run(`cd ${tmpDir} && git sparse-checkout set ${args.example}`);
-    // Copy out into a new directory within current working directory.
-    await run(`cp -R ${tmpDir}/${args.example} ${dirName}`);
+
+    // move out into a new directory.
+    await fs.moveSync(`${tmpDir}/${args.example}`, dirName);
+
     // Delete the clone.
-    await run(`rm -rf ${tmpDir}`);
+    await fs.removeSync(tmpDir);
 
     // Project Setup
     await installDependencies(dirName);
     await initGit(dirName);
   } catch (err) {
-    console.error(err);
-    if (fs.existsSync(dirName)) await run(`rm -rf ${dirName}`);
-    if (fs.existsSync(tmpDir)) await run(`rm -rf ${tmpDir}`);
+    if (fs.existsSync(dirName)) await fs.remove(dirName);
```

Comment:
Why did you move/remove the error logging?

Reply:
Also, FWIW, `run` was made into an async function so we didn't get into callback hell. If the fs lib has sync methods that don't use callbacks, I wouldn't go async here.

Reply:
Removing Error logging was a mistake,I forgot to put it back while trying to test something. My bad